config_probe/__init__.py

```diff
@@ -5,6 +5,13 @@
 import yaml
 from munch import munchify
 
+
+class ConfigNotFound(AttributeError):
+    pass
+
+
+from config_probe.munch_wrapper import MunchWrapper
```

Comment:
I would prefer that we avoid mutual imports.

Let's put ConfigNotFound exception in a new module called config_probe.exceptions.  This way, config_probe.__init__ will import it and config_probe.munch_wrapper will import it.  config_probe.munch_wrapper will no longer require to import config_probe.__init__